src/invoices/InvoiceLineDetails/InvoiceLineDetails.js

```diff
@@ -63,14 +74,24 @@ class InvoiceLineDetails extends Component {
 
   unmountDeleteConfirm = () => this.setState({ showConfirmDelete: false });
 
+  areTagsEnabled = () => {
+    const { resources } = this.props;
+    const tagSettings = (resources.configTags || {}).records || [];
```

Comment:
we don't use that patter to get values. Usually we use `get` from lodash

Reply:
Updated